master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,54 @@
+---
+title: IP Pool Manipulation
+---
+This document describes how to make common modifications to Calico's
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.  The
+`-o yaml` will output in a format that can be captured, manipulated, and then
+used to apply the desired changes.
+
+```shell
+$ calicoctl get ipPool -o yaml
+- apiVersion: v1
+  kind: ipPool
+  metadata:
+    cidr: 192.168.0.0/16
+  spec: {}
+```
+
+## Enabling IPIP and NAT Outgoing on a pool
+
+### 1. Retrieve current IP Pool config
+
+```shell
+$ calicoctl get ipPool 192.168.0.0/16 -o yaml > pool.yaml
```

Comment:
Since this is supposed to be a "general" guide using 192 as an example, can you remove the specific pool from this line?
```
$ calicoctl get ipPool -o yaml > pool.yaml
```